test/acceptance/editors-integration-test.js

```diff
@@ -66,4 +66,362 @@ describe('editors integration', function () {
       expect(template).toBe('<button type="button"></button>');
     });
   });
+
+  describe('with embedded templates', function () {
```

Comment:
Can you add a test for the example I posted in another comment:

```js
export default class Foo extends Service {
  indentedThing = stripIndent`
    Hahaha, this is just a plain string. Definitely not a template.
  `
}
```

The test should assert that the contents of `stripIndent` are **not** linted (I expect this test to fail with the current implementation).

Reply:
Added 'does not parse regular strings as a template' to prove this is actually failing as _not_ expected.